sites/partners/pages/application/[id]/applicationsCols.tsx

```diff
@@ -0,0 +1,106 @@
+import Link from "next/link"
+
+import { t } from "@bloom-housing/ui-components"
+import { convertDataToPst } from "../../../lib/helpers"
+import { ApplicationSubmissionType } from "@bloom-housing/backend-core/types"
+
+export const getCols = () => [
+  {
+    headerName: t("application.details.number"),
+    field: "id",
+    sortable: false,
+    filter: false,
+    resizable: true,
+    pinned: "left",
+    headerCheckboxSelection: true,
+    checkboxSelection: true,
+    cellRendererFramework: ({ data }) => {
+      if (!data?.id && !data?.confirmationCode) return ""
+      return <Link href={`/application/${data.id}`}>{data.confirmationCode || data.id}</Link>
+    },
+    width: 180,
+  },
+
+  {
+    headerName: t("application.name.firstName"),
+    field: "applicant.firstName",
+    sortable: false,
+    filter: false,
+    resizable: true,
+    unSortIcon: true,
+  },
+  {
+    headerName: t("application.name.lastName"),
+    field: "applicant.lastName",
+    sortable: false,
+    filter: false,
+    resizable: true,
+    unSortIcon: true,
+  },
+  {
+    headerName: t("applications.table.primaryDob"),
+    field: "applicant",
+    sortable: false,
+    filter: false,
+    resizable: true,
+    unSortIcon: true,
+    valueFormatter: ({ value }) => {
+      if (!value) return ""
+
+      const isValidDOB = !!value?.birthMonth && !!value?.birthDay && value?.birthYear
+
+      return isValidDOB ? `${value.birthMonth}/${value.birthDay}/${value.birthYear}` : ""
+    },
+    width: 120,
+  },
+  {
+    headerName: t("application.details.type"),
+    field: "submissionType",
+    sortable: false,
+    filter: false,
+    resizable: true,
+    unSortIcon: true,
+    valueGetter: ({ data }) => {
+      return data.submissionType === ApplicationSubmissionType.electronical
+        ? t("application.details.submissionType.digital")
+        : t("application.details.submissionType.paper")
+    },
+    width: 130,
+  },
+  {
+    headerName: t("applications.table.applicationSubmissionDate"),
+    field: "submissionDate",
+    sortable: false,
+    filter: false,
+    resizable: true,
+    valueGetter: ({ data }) => {
+      if (!data?.submissionDate) return ""
+
+      const { submissionDate } = data
+
+      const dateTime = convertDataToPst(
```

Comment:
Definitely not something we should handle in this PR, but we might want to keep in mind that pacific time is hard coded a few places. If we get more jurisdictions outside of the west coast or wanted to port this over to Detroit we would have to make code changes